resources/istio/values.yaml

```diff
@@ -1,5 +1,7 @@
 # Common settings.
 global:
+  isLocalEnv: false
```

Comment:
You cannot modify anything in istio chart as its raw copy from istio official release. Please, revert your changes and bring overrides back.

Reply:
this value is not needed here, in istio charts we do not use it as they raw from release like Szymon mentioned